Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.7.3'
+ruby '2.7.0'
 
```

Comment:
[Required]
Please, don't change the version of the ruby to 2.7.0 because we made an agreement earlier that we will use the 2.7.3 version.